bedrock/mozorg/templates/mozorg/internet-health/open-innovation.html

```diff
@@ -126,12 +126,15 @@ <h3 class="topic-action-title">{{ _('How you can support open source') }}</h3>
             <li>
               <h4>{{ _('Contribute to a project') }}</h4>
               <p>
-              {% trans contribute=url('mozorg.contribute.index'),
-                       open_hatch='https://openhatch.org/'
-              %}
-                Contribute to an existing open source project. Become a <a href="{{ contribute }}">Mozillian!</a>
-                Or use <a href="{{ open_hatch }}">OpenHatch</a> to match your skillset to a project.
-              {% endtrans %}
+              {% if l10n_has_tag('open-innovation-update-20171130') %}
+                  {% trans contribute=url('mozorg.contribute.index') %}
+                     Contribute to an existing open source project. Become a <a href="{{ contribute }}">Mozillian!</a>
+                  {% endtrans %}
+              {% else %}
+                  {% trans contribute=url('mozorg.contribute.index'),
+                           open_hatch='https://openhatch.org/' %}
+                     Contribute to an existing open source project. Become a <a href="{{ contribute }}">Mozillian!</a> Or use <a href="{{ open_hatch }}">OpenHatch</a> to match your skillset to a project.
+              {% endif %}
```

Comment:
Missing the closing `{% endtrans %}` before this `{% endif %}`. After that fix, I think we'll be good!

Reply:
Oops! You're totally right, of course. Thank you!

Reply:
All fixed, I believe.